extension/src/format/parser.ts

```diff
@@ -328,7 +337,9 @@ export namespace LispParser {
                             }
                         } else {
                             if (curr === '\r' || curr === '\n') {
-                                result.atoms.push(new LispAtom(grpStart.line, grpStart.character, temp));
+                                if (temp !== '') {
```

Comment:
is this a fix of some bug?

Reply:
If I remember right, this fixed an edge case where there was no blank line at the end of the document.